api/SVGGraphicsElement.json

```diff
@@ -5,10 +5,10 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGGraphicsElement",
         "support": {
           "chrome": {
-            "version_added": true
+            "version_added": "30"
```

Comment:
Chromium 30 matches the date when https://chromium.googlesource.com/chromium/src/+/1c0121db5bdec1c757a78a9eb7b2888985809cd8 was landed.